modules/comparison-of-solutions-to-provision-node-local-storage.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent_storage_local/overview-and-comparison-of-local-storage-options
+
+:_mod-docs-content-type: CONCEPT
+[id="comparison-of-solutions-to-provision-node-local-storage_{context}"]
+= Comparison of LVM Storage, LSO, and HPP
+
+[id="comparing-storage-types"]
+== Comparison of the support for storage types and filesystems
+
+The following table compares the support for storage types and filesystems provided by {lvms}, Local Storage Operator (LSO), and HostPath Provisioner (HPP) to provision local storage:
+
+.Comparison of the support for storage types and filesystems
+[cols="6a,5a,5a,5a",options="header"]
+|====
+
+|Functionality|LVM Storage|LSO |HPP
+
+|Support for block storage |Yes |Yes |No
+
+|Support for file storage |Yes |Yes |Yes
+
+|Support for object storage ^[1]^ |No |No |No
+
+|Available filesystems |`ext4`, `xfs` |`ext4`, `xfs` |Any mounted system available on the node is supported.
+
+|====
+[.small]
+--
+1. None of the solutions ({lvms}, LSO, and HPP) provide support for object storage. Therefore, an S3 object storage solution, such as `MultiClusterGateway` from the Red Hat OpenShift Data Foundation, is needed. All of the solutions can serve as underlying storage providers for the S3 object storage solutions.
+--
+
+[id="comparing-core-functionalities"]
+== Comparison of the support for core functionalities
+
+The following table compares how {lvms}, Local Storage Operator (LSO), and HostPath Provisioner (HPP) support core functionalities for provisioning local storage:
+
+.Comparison of the support for core functionalities
+[cols="6a,5a,5a,5a",options="header"]
+|====
+
+|Functionality|LVM Storage|LSO |HPP
+
+|Support for automatic file system formatting |Yes |Yes |N/A
+
+|Support for dynamic provisioning |Yes |No |No
+
+|Support for using software Redundant Array of Independent Disks (RAID) arrays 
+|Yes 
+
+Supported on 4.15 and higher versions.
+
+|Yes |Yes
+
+|Support for transparent disk encryption |Yes 
+
+Supported with 4.16 and higher versions.
+
+|Yes |Yes
+
+|Support for volume based disk encryption |No |No |No
+
+|Support for disconnected installation |Yes |Yes |Yes
+
+|Support for PVC expansion |Yes |No |No
+
+|Support for volume snapshots and volume clones |Yes |No |No
+
+|Support for thin provisioning |Yes
+
+Devices are thin-provisioned by default.
+
+|Yes
+
+You can configure the devices to point to thin-provisioned volumes
+
+|Yes
+
+You can configure a path to point to thin-provisioned volumes.
+
+|Support for automatic disk discovery and setup |Yes
+
+Automatic disk discovery is available during installation and runtime. You can also dynamically add the disks to the storage classes to increase the storage capacity.
```

Comment:
I'm not the LVMS expect, could anyone from LVMS team to confirm:

You can also dynamically add the disks to the `storage classes` to increase the storage capacity.
or should be:
You can also dynamically add the disks to the `LVMCluster` to increase the storage capacity.

Reply:
Hi @duanwei33 
Yes, that's right! However, upon configuring the device classes (where we add disks) in the LVMCluster CR, LVM Storage automatically creates a storage class for each device class. These storage classes can be specified in the PVCs. 

I think saying "add the disks to LVMCluster CR" will be very specific. But using "storage classes" provides more context I guess. 

@jakobmoellerdev 
WDYT?

Reply:
I think @duanwei33 is correct, but you have a point. we can respecify to 

> You can also dynamically add the disks to LVMCluster to increase the storage capacity of existing storage classes.

Reply:
This sounds great!
Updated as per the suggestion.